stdlib/public/SDK/Foundation/NSStringAPI.swift

```diff
@@ -636,7 +636,8 @@ extension String {
   public func getCString(
     _ buffer: inout [CChar], maxLength: Int, encoding: Encoding
   ) -> Bool {
-    return _ns.getCString(&buffer, maxLength: min(buffer.count, maxLength),
+    return _ns.getCString(&buffer,
```

Comment:
Please, no endline layout (all opening delimiters whose matching close does not fit on the same line should be followed  by newline)